python/pyarrow/_csv.pyx

```diff
@@ -342,9 +342,9 @@ cdef class ConvertOptions:
     ----------
     check_utf8 : bool, optional (default True)
         Whether to check UTF8 validity of string columns.
-    column_types: dict, optional
-        Map column names to column types
-        (disabling type inference on those columns).
+    column_types: pa.Schema or dict, optional
+        Explicitly map column names to column types. Passing this argument
```

Comment:
I'm not sure how to clarify it better.

Reply:
I'm surprised: was the original spelling not clear?